backend/src/main/resources/application.properties

```diff
@@ -1 +1,17 @@
+#H2
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2-console/
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.jpa.hibernate.naming.physical-strategy=org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
 
+
+#Flyway
+flyway.user=sa
+flyway.password=
+flyway.url=jdbc:h2:mem:testdb
+flyway.locations=filesystem:db/migration
+spring.flyway.baseline-on-migrate = true
```

Comment:
wrzuciłbym to wyżej, do 10 lini zeby springowy setup był razem a oddzielnie flyway